avocado/plugins/jobs.py

```diff
@@ -78,8 +78,7 @@ def _save_stream_to_file(self, stream, filename):
         """
         dirname = os.path.dirname(filename)
         if not os.path.isdir(dirname):
-            LOG_UI.error("%s does not exist. Exiting...", dirname)
-            return exit_codes.AVOCADO_GENERIC_CRASH
+            os.makedirs(dirname, exist_ok=True)
```

Comment:
What is the reason to set `exist_ok=True` here if you checked it in the previous `if`?

Reply:
You are absolutely right. Missed that, probably without coffe at the time.

Reply:
I know what you mean :D

Reply:
@willianrampazzo since it was a small fix, I just force pushed here.

Reply:
Cool, thanks!